README.md

```diff
@@ -105,7 +105,7 @@ const CustomPropsBreadcrumb = ({ someProp }) => (
 const routes = [
   { path: '/users/:userId', breadcrumb: DynamicUserBreadcrumb },
   { path: '/example', breadcrumb: 'Custom Example' },
-  { path: '/custom-props, breadcrumb: CustomPropsBreadcrumb, props: { someProp: 'Hi' }}, 
```

Comment:
Should keep the `props: { ... }` example here - it's a pretty common pattern for people to want to inject some extra data into certain breadcrumbs

Reply:
Oh, nope my bad!